templates/partials/configuration_upgrade.html

```diff
@@ -34,22 +34,23 @@
       <p ng-hide="versions.releases.length" translate>instance.upgrade.no_new_releases</p>
       <ul ng-show="versions.releases.length" class="table table-striped">
         <li ng-repeat="release in versions.releases" class="row">
-          <div class="col-xs-5"><code>{{release.id}}</code></div>
+          <div class="col-xs-5"><span class="label label-info ">{{release.id}}</span></div>
           <div class="col-xs-4" ng-bind-html="release.value.build_time | simpleDate"></div>
-          <div class="col-xs-3"><button ng-click="upgrade(release.id)" class="btn btn-primary" translate>upgrade</button></div>
+          <div class="col-xs-3"><button ng-click="upgrade(release.id)" class="btn btn-primary" translate>instance.upgrade.install</button></div>
         </li>
       </ul>
     </div>
+    <div class="row mb-50"></div>
```

Comment:
If the above suggestion works then you won't need this element.

Reply:
A "grey line" separator was specifically requested in the issue. I changed it to a hr.